sample-lnd.conf

```diff
@@ -328,132 +386,140 @@
+; max-channel-fee-allocation=0.5
 
 ; The maximum fee rate in sat/vbyte that will be used for commitments of
 ; channels of the anchors type. Must be large enough to ensure transaction
-; propagation (default: 10)
-; max-commit-fee-rate-anchors=5
+; propagation 
+; max-commit-fee-rate-anchors=10
 
 ; A threshold defining the maximum amount of dust a given channel can have
 ; after which forwarding and sending dust HTLC's to and from the channel will
-; fail. This amount is expressed in satoshis. (default: 500000)
-; dust-threshold=1000000
+; fail. This amount is expressed in satoshis.
+; dust-threshold=500000
 
 ; If true, lnd will abort committing a migration if it would otherwise have been
 ; successful. This leaves the database unmodified, and still compatible with the
 ; previously active version of lnd.
-; dry-run-migration=true
+; dry-run-migration=false
 
 ; If true, option upfront shutdown script will be enabled. If peers that we open
 ; channels with support this feature, we will automatically set the script to
 ; which cooperative closes should be paid out to on channel open. This offers the
 ; partial protection of a channel peer disconnecting from us if cooperative
 ; close is attempted with a different script.
-; enable-upfront-shutdown=true
+; enable-upfront-shutdown=false
 
 ; If true, spontaneous payments through keysend will be accepted.
 ; This is a temporary solution until AMP is implemented which is expected to be soon.
 ; This option will then become deprecated in favor of AMP.
-; accept-keysend=true
+; accept-keysend=false
 
 ; If non-zero, keysend payments are accepted but not immediately settled. If the
 ; payment isn't settled manually after the specified time, it is canceled
 ; automatically. [experimental]
-; keysend-hold-time=true
+; Default:
+;   keysend-hold-time=0s
+; Example:
+;   keysend-hold-time=2s
 
 ; If true, spontaneous payments through AMP will be accepted. Payments to AMP
 ; invoices will be accepted regardless of this setting.
-; accept-amp=true
+; accept-amp=false
 
 ; If true, we'll attempt to garbage collect canceled invoices upon start.
-; gc-canceled-invoices-on-startup=true
+; gc-canceled-invoices-on-startup=false
 
 ; If true, we'll delete newly canceled invoices on the fly.
-; gc-canceled-invoices-on-the-fly=true
+; gc-canceled-invoices-on-the-fly=false
 
 ; If true, our node will allow htlc forwards that arrive and depart on the same
 ; channel.
-; allow-circular-route=true
+; allow-circular-route=false
 
 ; Time in milliseconds between each release of announcements to the network
-; trickledelay=180000
+; trickledelay=90000
 
 ; The number of peers that we should receive new graph updates from. This option
-; can be tuned to save bandwidth for light clients or routing nodes. (default: 3)
-; numgraphsyncpeers=9
+; can be tuned to save bandwidth for light clients or routing nodes. 
+; numgraphsyncpeers=3
 
 ; If true, lnd will start the Prometheus exporter. Prometheus flags are
```

Comment:
fix it.
Btw: 
The ends of the groups are not uniformly formatted. I will bring all ends to the following format: 2 line breaks, name of the new group, 1 line break.

Also, while scrolling, I noticed a few blank lines with a semicolon at the beginning. I will also delete the semicolons so that it is consistent.

Reply:
👌